.golangci.yml

```diff
@@ -1,35 +1,65 @@
-service:
-  golangci-lint-version: 1.23.0
-
 run:
   timeout: 5m
+  modules-download-mode: readonly
 
 linters-settings:
+  goconst:
+    min-len: 2
+    min-occurrences: 2
   gofmt:
     simplify: true
+  goimports:
+    local-prefixes: github.com/mattermost/mattermost-plugin-github
+  golint:
+    min-confidence: 0
   govet:
     check-shadowing: true
     enable-all: true
+  misspell:
+    locale: US
 
 linters:
   disable-all: true
   enable:
+    - bodyclose
     - deadcode
+    - errcheck
+    - goconst
+    - gocritic
     - gofmt
+    - goimports
+    - golint
+    - gosec
     - gosimple
     - govet
     - ineffassign
+    - interfacer
+    - misspell
+    - nakedret
+    - staticcheck
     - structcheck
+    - stylecheck
+    - typecheck
     - unconvert
     - unused
     - varcheck
-    # TODO: enable this later
-    # - errcheck
+    - whitespace
 
 issues:
   exclude-rules:
-    - linters:
-      # ignore unused warning for manifest
-      - varcheck
-      - deadcode
-      text: "manifest"
+    - path: server/manifest.go
+      linters:
+        - deadcode
+        - unused
+        - varcheck
+    - path: server/configuration.go
+      linters:
+        - unused
+    - path: _test\.go
+      linters:
+        - bodyclose
+        - goconst
+        - scopelint # https://github.com/kyoh86/scopelint/issues/4
+    - path: server/webhook.go
+      linters:
+        - goconst
```

Comment:
There are so many action types like `"opened" ` in `server/webhook.go` that refactoring them into `const`'s did make the code more readable.